internal/isoutil/isoutil.go

```diff
@@ -308,6 +312,54 @@ func WriteHeader(ignitionOffsetInfo, ramDiskOffsetInfo *OffsetInfo, isoPath stri
 	return nil
 }
 
+func ReadHeader(isoPath string) (int64, int64, error) {
```

Comment:
This is rhcos iso specific, so I think it should be in the editor.

Reply:
right, moved.